rr_computer_vision/src/computer_vision_main.cpp

```diff
@@ -20,8 +18,9 @@ int main(int argc, char** argv)
   ROS_INFO("Starting Computer Vision Node");
   
   ComputerVision computer_vision(nh);
+  TrafficLightProcessor tlproc(nh);
```

Comment:
Comment this out before merging, don't want to add overhead for other people when the pull develop into their branch and end up launching all the traffic light processing. Once all the computer vision are into develop we will organize to initialize objects more intelligently based off the race type